spec/concepts/api/v1/users/lib/service/session_token_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Api::V1::Users::Lib::Service::SessionToken do
   describe '.create' do
-    subject(:service) { described_class.create(params) }
+    subject(:service) { described_class.create(**params) }
```

Comment:
For what double splat??? It's do the same.

Reply:
same